src/components/awards-entry.svelte

```diff
@@ -0,0 +1,41 @@
+<script lang="ts">
+  import { Button } from '@significa/svelte-ui';
+  import { getImageAttributes } from '$lib/utils/cms';
+  import type { AssetStoryblok } from '$types/bloks';
+
+  export let linkHref: string | undefined;
+  export let image: AssetStoryblok | undefined;
+  export let label: string | undefined;
+  export let name: string | undefined;
+  export let project: string | undefined;
+  export let linkText: string | undefined;
+</script>
+
+<div class="flex w-full flex-col-reverse items-center lg:flex-row">
+  <div class="mb-4 flex w-full items-center lg:mb-0">
+    {#if image?.filename}
+      {@const { alt, src, width, height } = getImageAttributes(image, {
+        size: [120, 0]
+      })}
+      <img class="mr-2 h-auto w-14 rounded-xs bg-background-offset" {src} {alt} {width} {height} />
+    {/if}
+    <div class="ml-4 flex-col">
+      <p class="text-base font-semibold text-foreground-secondary">
+        {label}
+      </p>
+      <p class="text-base font-semibold">{name}</p>
+    </div>
+  </div>
+  <div class="mb-4 w-full lg:mb-0">
+    <p class="text-3xl font-semibold">{project}</p>
+  </div>
+</div>
+<div class="w-1/3">
+  {#if linkHref}
```

Comment:
`href` isn't being used

Reply:
It wasn't being used because the entire div is wrapped in a link that you can click to show more details. That verification was only there not show the button if the link didn't exist. I've added the link href to the button anyway, so the component is more flexible